app/models/company.rb

```diff
@@ -1,10 +1,15 @@
 class Company < ApplicationRecord
   extend FriendlyId
-  friendly_id :official_name, use: :slugged
+  friendly_id :official_name, use: :slugged, slug_column: :subdomain
+
+  SUBDOMAIN_REGEXP = /\A(?!(http|https|www)\z)(\A[\w\-_]+)\z/
```

Comment:
What do you think about validating subdomain of the company by this SUBDOMAIN_REGEXP